common/pb/moderator_commands.proto

```diff
@@ -17,3 +18,10 @@ message Command_BanFromServer {
     optional string visible_reason = 5;
     optional string clientid = 6;
 }
+
+message Command_GetBanHistory {
+    extend ModeratorCommand {
+        optional Command_GetBanHistory ext = 1001;
+    }
+    optional string user_name = 1;
```

Comment:
Yeah we dont want to make it required as a ban could be placed with out the username.  Ill check out what happens if the name is blank in the current code.  Im expecting it to return bans that don't have username's associated with them (which can be made).  It wouldnt return all the bans.